.github/workflows/docker.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: ${{ github.repository }}:2.6.7
+          tags: ${{ github.repository }}:2.7
```

Comment:
Is any repo still stuck on 2.6.x?

Reply:
Any repo on 2.6.x can probably just bump to 2.7.x? (Just guessing that it's fine but no idea)

Reply:
The 2.6.7 docker image will still remain, and we pull it with the use of tags so I don't _think_ this should break anything

Reply:
Negative. All of rewind is using 2.6.7 (coupled with EB) so we should keep that around so we can continue to get updates for the 2.6 line.
Maybe a matrix build or multiple dockerfiles is needed?

This image was created specifically for buddy build jobs in order to bundle package a rails project in order to match the arch of elasticbeanstalk.  Is ultron using elasticbeanstalk? 

if so, it is bound to ruby 2.7.4
https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platforms-supported.html#platforms-supported.ruby no?

Reply:
Is the rewind-app able to bump the EB runtime? 

If that's a PITA, might be nice to keep 2.6.x around in case there is some CVE patch.